docs/docs/30-administration/11-forges/60-bitbucket_datacenter.md

```diff
@@ -0,0 +1,95 @@
+# Bitbucket Datacenter / Server
+
+:::warning
+Woodpecker comes with experimental support for Bitbucket Datacenter / Server, formerly known as Atlassian Stash.
+:::
+
+To enable Bitbucket Server you should configure the Woodpecker container using the following environment variables:
+
+```diff
+# docker-compose.yml
+version: '3'
+
+services:
+  woodpecker-server:
+    [...]
+    environment:
+      - [...]
++     - WOODPECKER_BITBUCKET_DC=true
++     - WOODPECKER_BITBUCKET_DC_GIT_USERNAME=foo
++     - WOODPECKER_BITBUCKET_DC_GIT_PASSWORD=bar
++     - WOODPECKER_BITBUCKET_DC_CLIENT_ID=xxx
++     - WOODPECKER_BITBUCKET_DC_CLIENT_SECRET=yyy
++     - WOODPECKER_BITBUCKET_DC_URL=http://stash.mycompany.com
+
+  woodpecker-agent:
+    [...]
+```
+
+## Service Account
+
+Woodpecker uses `git+https` to clone repositories, however, Bitbucket Server does not currently support cloning repositories with an OAuth token. To work around this limitation, you must create a service account and provide the username and password to Woodpecker. This service account will be used to authenticate and clone private repositories.
```

Comment:
Depending on the configuration Bitbucket also supports git+ssh. I guess the original implementation chose http because if you have access to the API of Bitbucket you should also have access clone using http.

Reply:
If http works we can stick to it for now and adjust later on